### ISSUE TITLE
Support `<scheme>.sub_plain` for OpenFHE backend

### ISSUE THREAD

**Issue by AlexanderViand-Intel** [google/heir#1200]
Currently, we have `bgv.sub_plain` and `ckks.sub_plain` but no way to target them to OpenFHE

**Comment by asraa** [comment]
I think OpenFHE's EvalMul and EvalAdd are overloaded to support ciphertext-ciphertext and ciphertext-plaintext

**Comment by AlexanderViand-Intel** [comment]
> I think OpenFHE's EvalMul and EvalAdd are overloaded to support ciphertext-ciphertext and ciphertext-plaintext

Yep, and those the pipeline already supports. But I'm not sure its supported for EvalSub in OpenFHE? If not, that just means we need to translate it to "negate plaintext, then add" (assuming that's the direction the op is defined in, which I don't recall right now 🙈 )

**Comment by asraa** [comment]
Oh! Gotcha - I think you're right about that and having some kind of rewrite/canonicalization pattern.
